src/api/IBMiContent.ts

```diff
@@ -696,7 +696,7 @@ export default class IBMiContent {
   }
 
   async streamfileResolve(names: string[], directories: string[]): Promise<string | undefined> {
-    const command = `for f in ${directories.flatMap(dir => names.map(name => path.posix.join(dir, name))).join(` `)}; do if [ -f $f ]; then echo $f; break; fi; done`;
+    const command = `for f in ${directories.flatMap(dir => names.map(name => `"${path.posix.join(dir, name)}"`)).join(` `)}; do if [ -f $f ]; then echo $f; break; fi; done`;
```

Comment:
@worksofliam I added a test for the `streamfileResolve` function and found an error: `-bash: [: too many arguments`

You need to encapsulate $f in the [] test like this:

```suggestion
    const command = `for f in ${directories.flatMap(dir => names.map(name => `"${path.posix.join(dir, name)}"`)).join(` `)}; do if [ -f "$f" ]; then echo $f; break; fi; done`;
```